README.md

```diff
@@ -17,7 +17,7 @@ Command-line AirPlay video client for Apple TV
 Requirements
 -------------------------------------------------------------------------------
 
-- OS X or Ubuntu
+- OS X, Ubuntu, Arch Linux ([nss-mdns](https://www.archlinux.org/packages/extra/x86_64/nss-mdns/) package is required)
```

Comment:
Could you please write detailed command to install like [Ubuntu](https://github.com/Tomohiro/airplayer#ubuntu)?